src/main/java/frc/robot/subsystems/intakeLimelight/IntakeLimelightConstants.java

```diff
@@ -0,0 +1,5 @@
+package frc.robot.subsystems.intakeLimelight;
+
+public class IntakeLimelightConstants {
+  
```

Comment:
Camera pitch goes here probably

Reply:
Done. I think it's 0.0deg, correct me if I'm wrong

Reply:
I don't know, didn't see it in the sheet